base/src/main/java/org/robojackets/apiary/base/ui/theme/Settings.kt

```diff
@@ -0,0 +1,14 @@
+package org.robojackets.apiary.base.ui.theme
+
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+
+// @Composable
+// private fun Settings() {
+// // TODO
+// }
+
+@Composable
+fun SettingsScreen() {
+    Text("Hello")
```

Comment:
What are you planning on putting here?

Reply:
Logout button, build info, that kinda thing